src/cloco-client.ts

```diff
@@ -33,22 +33,25 @@ export class ClocoClient {
 
     constructor(options: IOptions) {
 
-      try {
-        this.options = options;
-
-        // initialize components.
-        Logger.init(this.options);
-        Cache.init();
-        this.onCacheExpired.subscribe((client: ClocoClient, item: CacheItem) => {
-          this.loadConfigurationObjectWrapperFromApi(item.key);
-        });
-
-        // set default values.
-        Settings.setDefaults(this.options);
-      } catch (e) {
-        Logger.log.error(e, "ClocoClient()");
-        throw e;
-      }
+        try {
+            this.options = options;
+
+            // initialize components.
+            Logger.init(this.options);
+            Cache.init();
+            this.onCacheExpired.subscribe((client: ClocoClient, item: CacheItem) => {
+                this.loadConfigurationObjectWrapperFromApi(item.key);
+            });
+
+            // ensure the local cache folder is available
+            FileSystem.ensureDirectory(`${FileSystem.getUserHome()}/.cloco/cache`);
```

Comment:
Yes, all read/write is from user home, so the account should be able to have access.